app/modules/FrontModule/Registration/components/RegistrationForm/RegistrationForm.php

```diff
@@ -130,29 +126,47 @@ private function processForm(ArrayHash $values): void
 	{
 		$values->email = $values->liame;
 
-		if ($values['vegetarian']){
-			$values['vegetarian'] = 'yes';
-		} else {
-			$values['vegetarian'] = 'no';
-		}
-		if ($values['invoice']){
-			$values['invoice'] = 'yes';
-		} else {
-			$values['invoice'] = 'no';
-		}
-
-		$participant = new Registration(2020, $values->name, $values->nickname, $values->email, $values->phone, $values->arrival, $values->invoice, $values->companyid, $values->vegetarian,
-			$values->skills, $values->tshirt, $values->presentation, $values->note);
+		$values['vegetarian'] = $values['vegetarian'] ? 'yes' : 'no';
+
+		$values['invoice'] = $values['invoice'] ? 'yes' : 'no';
+
+		$participant = new Registration(
```

Comment:
tohle je divne jak je to na kazdem radku ? nezda se ti to ?

Reply:
Mě to třeba vyhovuje, přijde mi to daleko víc čitelné než jeden ultradlouhý řádek (ve výsledku stejně pak IDE zalamuje řádky (alespoň to tak mám já) a když to dělám nativně přes codestyle, přecijen je to naformátované líp

Reply:
me IDE radky nezalamuje, takze to mam na jedne dlouhe radce a nemam s tim problem. Nejsem pedant na 80 znaku na radek ... kort u takovehle veci... asi osobni pohled kazdeho programatora :(  na tohle nebude jedna spravna pravda

Reply:
To asi ne, mám to pravidlo vyhodit?

Reply:
ano prosim, vyhodit prozatim pravidlo na tenhle prevod dlouhych radku na vice radku